### PR TITLE
Cfuzz

### PR DESCRIPTION
**Pull request by zerokay**
cfuzz experiment (compare against core fuzzers)

### CODE REVIEW

**Comment by zerokay** [comment]
hi, @Alan32Liu I have successfully signed google-cla. However,  I received the message: The modified files do not match the file filter. What should I do next? Thanks.

**Comment by Alan32Liu** [comment]
Would you mind making a trivial modification to [service/gcbrun_experiment.py](https://github.com/google/fuzzbench/blob/master/service/gcbrun_experiment.py)?
This will allow me to launch experiments in this PR before merging. Here is [an example to add a dummy comment](https://github.com/google/fuzzbench/pull/1818/files#diff-e870d8214beac56eaaeda2edc69de842f8382c428329dd8cf826a9cfa5edf7e9) : )
Thanks!

**Comment by zerokay** [comment]
> Would you mind making a trivial modification to [service/gcbrun_experiment.py](https://github.com/google/fuzzbench/blob/master/service/gcbrun_experiment.py)? This will allow me to launch experiments in this PR before merging. Here is [an example to add a dummy comment](https://github.com/google/fuzzbench/pull/1818/files#diff-e870d8214beac56eaaeda2edc69de842f8382c428329dd8cf826a9cfa5edf7e9) : ) Thanks!

Done

**Comment by Alan32Liu** [comment]
/gcbrun run_experiment.py -a --experiment-config /opt/fuzzbench/service/experiment-config.yaml --experiment-name 2023-07-26-cfuzz  --fuzzers cfuzz

**Comment by Alan32Liu** [comment]
> Done

The names of core fuzzers are not included in the command, because FuzzBench should be able to reuse past results.

**Comment by Alan32Liu** [comment]
Experiment data and results will be available at:
[The experiment data](https://storage.googleapis.com/fuzzbench-data/index.html?prefix=2023-07-26-cfuzz/).
[The experiment report](https://storage.googleapis.com/www.fuzzbench.com/reports/experimental/2023-07-26-cfuzz/index.html).

File: fuzzers/cfuzz/runner.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+FROM gcr.io/fuzzbench/base-image
+
+# This makes interactive docker runs painless:
+ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/out"
+#ENV AFL_MAP_SIZE=2621440
+ENV PATH="$PATH:/out"
+ENV AFL_SKIP_CPUFREQ=1
+ENV AFL_I_DONT_CARE_ABOUT_MISSING_CRASHES=1
+ENV AFL_TESTCACHE_SIZE=2
+RUN apt install -y unzip git gdb joe
```
Comment (Alan32Liu) [inline fuzzers/cfuzz/runner.Dockerfile:24]: According to the error log, this is the reason for the build failure.
Could you please add an `apt-get update` statement before this?
It's also better to replace `apt` with `apt-get`.

That solves the issue when I test it in the docker container.

**Review by Alan32Liu — CHANGES_REQUESTED** [review]

File: fuzzers/cfuzz/runner.Dockerfile (hunk 1 above)
Comment (zerokay) [reply fuzzers/cfuzz/runner.Dockerfile]: Done

**Comment by Alan32Liu** [comment]
Experiment data and results will be available at:
[The experiment data](https://storage.googleapis.com/fuzzbench-data/index.html?prefix=2023-07-27-cfuzz/).
[The experiment report](https://storage.googleapis.com/www.fuzzbench.com/reports/experimental/2023-07-27-cfuzz/index.html).

**Comment by zerokay** [comment]
> Experiment data and results will be available at: [The experiment data](https://storage.googleapis.com/fuzzbench-data/index.html?prefix=2023-07-27-cfuzz/). [The experiment report](https://storage.googleapis.com/www.fuzzbench.com/reports/experimental/2023-07-27-cfuzz/index.html).

I have made the following settings in the experience-requests.yaml to compare with these fuzzers. 

> - experiment: 2023-07-23-cfuzz
>   description: "cfuzz experiment (compare against core fuzzers)"
>   fuzzers:
>     - afl
>     - aflfast
>     - aflplusplus
>     - aflsmart
>     - entropic
>     - eclipser
>     - fairfuzz
>     - honggfuzz
>     - lafintel
>     - libfuzzer
>     - mopt
>     - cfuzz

But I didn't see this comparison in the report.

**Comment by Alan32Liu** [comment]
The base fuzzer data should be added once the experiment completes.

**Comment by zerokay** [comment]
Thanks! @Alan32Liu

**Pull request closed without merging**